aten/src/ATen/native/quantized/cpu/qconv.cpp

```diff
@@ -620,16 +633,27 @@ class QConvInt8 final {
         output_zero_point,
         c10::nullopt);
 
+    qnnpack::conv_param_t conv_p(
+        {kernel_w, kernel_h},
+        {stride_w, stride_h},
+        {dilation_w, dilation_h},
+        {pad_h, pad_w, pad_h, pad_w},
+        groups,
+        C,
+        M,
+        (uint8_t*)pack_data.w_zero_points.data_ptr<c10::quint8>(),
```

Comment:
please avoid using c style casts

Reply:
This is from the original code.

Reply:
But besides that do you mean to suggest use static_cast<uint8_t*>?

Reply:
I think we use reinterpret_cast when accessing tensor data in most places.